force-app/main/default/classes/SOQL_Test.cls

```diff
@@ -825,61 +855,61 @@ private class SOQL_Test {
     @IsTest
     static void inlcudesAll() {
         // Setup
-        List<String> roles = new List<String>{ 'Business User', 'Decision Maker' };
+        List<String> industries = new List<String>{ 'Agriculture', 'Apparel' };
 
         // Test
-        SOQL builder = SOQL.of(AccountContactRelation.SObjectType)
-            .with(AccountContactRelation.Id)
-            .whereAre(SOQL.Filter.with(AccountContactRelation.Roles).includesAll(roles));
+        SOQL builder = SOQL.of(Account.SObjectType)
+            .with(Account.Id)
+            .whereAre(SOQL.Filter.with(Account.Industry).includesAll(industries));
 
         // Verify
         String soql = builder.toString();
-        Assert.areEqual('SELECT Id FROM AccountContactRelation WHERE Roles INCLUDES (\'Business User;Decision Maker\')', soql);
```

Comment:
INCLUDES działa tylko na multiselect pickliśice, a Industry nie jest multiselect :(

Reply:
AccountContactRelation is not enabled by default, test fails on new org

Reply:
Yes, you need to enabled it. There is no other multiselect picklist on standard object :(